components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/claims/impl/DefaultClaimHandler.java

```diff
@@ -1098,6 +1110,20 @@ private boolean enableMergingCustomClaimMappingsWithDefaultMappings() {
                 .isMergingCustomClaimMappingsWithDefaultClaimMappingsAllowed();
     }
 
+    /**
+     * Handle role app association in roles claim.
+     *
+     * @param appAssociatedRoles App associated roles.
+     * @param mappedAttrs Mapped claim attributes.
+     */
+    private void handleRoleAppAssoication(Map<String, String> mappedAttrs, String appAssociatedRoles) {
+
+        if (mappedAttrs.containsKey(getLocalGroupsClaimURI())) {
```

Comment:
assign getLocalGroupsClaimURI() to a variable and reuse

Reply:
Fixed with Fixed with https://github.com/wso2/carbon-identity-framework/pull/5070/commits/cf43977ae1c82e009dda795205668047624a0925